common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -1051,6 +1051,16 @@ public void run() {
         });
     }
 
+    @Override
+    public long submittedTasks() {
+        return PlatformDependent.currentProducerIndex(taskQueue);
```

Comment:
Do you think it matters that the semantics of producerIndex versus submittedTasks differ in the general case? In particular for the producerIndex `This value will normally indicate number of elements passed into the queue, but may under some circumstances be a derivative of that figure` ... means that some queue impls might cause incorrect values to be returned here (for example double the number of tasks actually submitted).

Reply:
That's something that has to be consider, but for the most used ones should be fine eg https://github.com/JCTools/JCTools/blob/master/jctools-core/src/main/java/org/jctools/queues/BaseMpscLinkedArrayQueue.java#L441